extensions/parquet/table/src/main/java/io/deephaven/parquet/table/location/ParquetColumnLocation.java

```diff
@@ -415,7 +431,7 @@ private static <ATTR extends Any, RESULT> ToPage<ATTR, RESULT> makeToPage(
             if (toPage == null) {
```

Comment:
See the comment above. I think maybe the right answer is "always make a `toPage` if we know how, but always make sure the type matches `pageType` and have a specific error in that case.